rr_iarrc/launch/lines_pcl_converter.launch

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <launch>
     <node pkg="rr_platform" type="image_transform" name="image_transform" output="screen">
-        <param name="transform_topics" value="/lines_detection_img"/>
+        <param name="transform_topics" value="/lines_detection_img /cone_detection_img"/>
```

Comment:
This should stay `/lines_detection_img `so we don't get confused later since the it is the line converter launch file.

Reply:
And same later when this was done.